dist/tools/compile_and_test_for_board/compile_and_test_for_board.py

```diff
@@ -33,36 +33,46 @@
 
 ```
 usage: compile_and_test_for_board.py [-h] [--applications APPLICATIONS]
-                                     [--applications-exclude
-                                      APPLICATIONS_EXCLUDE]
+                                     [--applications-exclude APPLICATIONS_EXCLUDE]
                                      [--no-test]
-                                     [--loglevel {debug,info,warning,error,
-                                                  fatal,critical}]
+                                     [--loglevel {debug,info,warning,error,fatal,critical}]
                                      [--incremental] [--clean-after]
+                                     [--compile-targets COMPILE_TARGETS]
+                                     [--test-targets TEST_TARGETS]
                                      [--jobs JOBS]
                                      riot_directory board [result_directory]
 
 positional arguments:
   riot_directory        RIOT directory to test
   board                 Board to test
-  result_directory      Result directory, by default "results"
+  result_directory      Result directory, by default "results" (default:
+                        results)
 
 optional arguments:
   -h, --help            show this help message and exit
   --applications APPLICATIONS
                         List of applications to test, overwrites default
-                        configuration of testing all applications
+                        configuration of testing all applications (default:
+                        None)
```

Comment:
Since you are disabling line length check with flake8 for the docstring, maybe put `(default: None)` on the same line.
Same for `loglevel` option.

Well, I tested putting the `default: None` on the same line locally and it breaks the test. After more tries, it seems that the test comparison is rather fragile regarding the comparison of the help message with the docstring. The help message text can be wrapped differently for different output lengths. What is the default value ? Is it always the same ?

Reply:
The output is generated by argparse.
It checks that it is the same as `./compile_and_test_for_board.py --help`.

Changing the output means changing `argparse.add_argument` help message.

Reply:
On my machine it is not wrapped differently whatever the output length is.

Reply:
I could also replace '(default: None)' by `(default: )` is wanted.

Reply:
Using default:None is ok for me. I don't what is the current usage for this.